backend/config/application.rb

```diff
@@ -21,7 +21,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Dotenv.load('/etc/.env', '.env')
+Dotenv.load('/etc/.env', '.env', '.env.local')
```

Comment:
Is .env.local something we'll have to set up locally that will be ignored in source control? if so just making sure i don't lose sight of this :D